docs/getting-started/installing-regression-games.mdx

```diff
@@ -4,6 +4,115 @@ sidebar_label: 'Installing Regression Games'
 
 # Installing Regression Games
 
-import Placeholder from '../partials/_placeholder.mdx';
+This page covers everything you need to know to add the Regression Games SDK to your Unity project and start exploring its features.
 
-<Placeholder />
\ No newline at end of file
+## Add the Regression Games SDK to Your Project
+
+Our core package is [publicly available on GitHub](https://github.com/Regression-Games/RGUnityBots).
+Open your Unity project and navigate to `Window` > `Package Manager`. From here, find the `Add package from git URL` and paste the following URL:
+
+```
+https://github.com/Regression-Games/RGUnityBots.git?path=src/gg.regression.unity.bots#v0.0.27
+```
+
+:::info
+- This package utilizes TextMeshPro. If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
+- This package installs the [Newtonsoft Json](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) dependency for serializing/deserializing Json message payloads.
+- If you get a note about Unity's new input system, click **Yes** and allow the editor to restart.
+:::
+
+![Add the Regression Games core SDK to your Unity project](img/installing-regression-games/installing-sdk.gif)
+
+## Add Extension Packages to Your Project
+
+We provide the following extensions for our core SDK to support common project dependencies.
+Each of these can be added to your project using a git url, similar to core package installation.
+
+### Animation Rigging Extension
+
+Adds support for Unity Animation Rigging.
```

Comment:
Maybe add a sentence about how we are using this? When, as a user, should I add this? We can potentially kick this to the docs of the part that uses Animation Rigging. Given that this is the first page, the more streamlined we can be the better.

Reply:
I actually don't know what happens if you don't have this extension in projects using Animation Rigging. @RG-nAmKcAz can you advise on what sort of data this captures / what the effects are of not including it?

Reply:
you get state data for animation rigging components in the state whereas you will not without it

Reply:
That's what I figured, wanted to make sure it wouldn't break anything if a project uses animation rigging but doesn't include this extension